core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/function/stringfunction/Position1Expression.kt

```diff
@@ -3,10 +3,11 @@ package ch.ergon.dope.resolvable.expression.unaliased.type.function.stringfuncti
 import ch.ergon.dope.resolvable.expression.TypeExpression
 import ch.ergon.dope.resolvable.expression.unaliased.type.function.FunctionExpression
 import ch.ergon.dope.resolvable.expression.unaliased.type.toDopeType
+import ch.ergon.dope.validtype.NumberType
 import ch.ergon.dope.validtype.StringType
 
 class Position1Expression(inStr: TypeExpression<StringType>, searchStr: TypeExpression<StringType>) :
-    FunctionExpression<StringType>("POSITION1", inStr, searchStr)
+    FunctionExpression<NumberType>("POSITION1", inStr, searchStr)
```

Comment:
Why aren't any tests affected? Shouldn't you add some then?

Reply:
The problem with the generic is that you can't realy test it exept if you write something like this:
`val actual: Position1Expression<NumberType> = position1("test", "test")` and we don't do that anywhere.
should i add that (i would need to add it to every function call that has uses a ValidType)?

Reply:
Well I would expect a test like:
`...where position1("lel","lol") + 4 < 7`
This ensures that it is a number-type
(I guess this bug came up, becuase such a test was missing)